react-checkout/src/components/Request.jsx

```diff
@@ -0,0 +1,98 @@
+import React, { Component } from 'react'
+import ReactDOM from 'react-dom'
+import hljs from 'highlight.js'
+
+class Request extends Component {
+  componentDidMount() {
+    this.highlightCode()
+  }
+
+  componentDidUpdate() {
+    this.highlightCode()
+  }
+
+  highlightCode() {
+    const domNode = ReactDOM.findDOMNode(this)
+    const nodes = domNode.querySelectorAll('pre code')
+
+    for (let i = 0; i < nodes.length; i++) {
+      hljs.highlightBlock(nodes[i])
+    }
+  }
+  render() {
+    const { checkout, host, checkoutObj } = this.props
+    const { type } = checkout
+
+    if (checkoutObj.data) {
+      if (checkoutObj.meta) delete checkoutObj.meta
+      const { attributes } = checkoutObj.data
+      delete attributes.available_payment_options
+      delete attributes.created
+      delete attributes.currency
+      delete attributes.donation
+      delete attributes.expires_on
+      delete attributes.payment_option_id
+      delete attributes.user
+      delete attributes.status
+      const { invoice, tickets } = attributes
+      if (invoice) {
+        delete invoice.reference
+        delete invoice.method
+        delete invoice.company
+        delete invoice.address_street
+        delete invoice.address_city
+        delete invoice.address_postalcode
+        delete invoice.address_region_id
+        delete invoice.address_country_id
+        delete invoice.phone
+        delete invoice.created
+      }
+      if (tickets) {
+        tickets.map(ticket => {
```

Comment:
Be aware that writing ES6 creates browser incompatibility (unless you're using something to compile the jsx to es5 that I'm not aware of). 
Depending on target audience, that may be something to think about.

Reply:
Using babel to compile into browser-friendly js. https://babeljs.io/

Reply:
@tmirmota Where do you define that? I was looking for mention of it but couldn't find anything.

Reply:
The app is built with create-react-app which comes pre built with the configuration for babel and webpack. If you want to see the config for each, you can run `npm run eject`. Keep in mind that once you eject you can not go back

Reply:
See docs on ejecting here: https://github.com/facebookincubator/create-react-app#converting-to-a-custom-setup

Reply:
Thanks, all I needed to know was where the mystery box was